roles/metrics/templates/grafana-dashboardDatasources.yaml

```diff
@@ -1,6 +1,17 @@
 apiVersion: v1
 data:
-  datasources.yaml: ewogICAgImFwaVZlcnNpb24iOiAxLAogICAgImRhdGFzb3VyY2VzIjogWwogICAgICAgIHsKICAgICAgICAgICAgImFjY2VzcyI6ICJwcm94eSIsCiAgICAgICAgICAgICJlZGl0YWJsZSI6IGZhbHNlLAogICAgICAgICAgICAibmFtZSI6ICJwcm9tZXRoZXVzIiwKICAgICAgICAgICAgIm9yZ0lkIjogMSwKICAgICAgICAgICAgInR5cGUiOiAicHJvbWV0aGV1cyIsCiAgICAgICAgICAgICJ1cmwiOiAiaHR0cDovL3Byb21ldGhldXMtazhzLmRvam90LW1vbml0b3Jpbmcuc3ZjOjkwOTAiLAogICAgICAgICAgICAidmVyc2lvbiI6IDEKICAgICAgICB9CiAgICBdCn0=
+  datasources.yaml: YXBpVmVyc2lvbjogMQpkYXRhc291cmNlczoKLSBuYW1lOiBQcm9tZXRoZXVzCiAgdHlwZTogcHJvbWV0aGV1cwogIHVybDogaHR0cDovL3Byb21ldGhldXMtazhzLmRvam90LW1vbml0b3Jpbmcuc3ZjLmNsdXN0ZXIubG9jYWw6OTA5MAogIGFjY2VzczogcHJveHkKICBpc0RlZmF1bHQ6IHRydWUKLSBuYW1lOiBMb2tpCiAgdHlwZTogbG9raQogIHVybDogaHR0cDovL2xva2kuZG9qb3QtbW9uaXRvcmluZy5zdmMuY2x1c3Rlci5sb2NhbDozMTAwCiAgYWNjZXNzOiBwcm94eQo=
+    # apiVersion: 1
```

Comment:
Should the lines 4..14 really be commented?

Reply:
Yes.

They're commented out just to show that base64 above refers to that snippet of configuration.

Reply:
It is a representation of base64 above.